src/screens/OtherProfile/index.js

```diff
@@ -628,14 +554,14 @@ const OtherProfile = () => {
           data={handleDataFeed()}
           onScroll={handleScroll}
           onRefresh={handleRefresh}
-          refreshing={loading}
+          refreshing={isLoading}
           onMomentumScrollEnd={(event) => {
             onWillSendViewPostTime(event, feeds);
             fetchNextFeeds(event);
           }}
           ListHeaderComponent={
             <>
-              {!initLoading ? (
+              {!isProfileFetching ? (
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `loading` to `isLoading` is more readable and understandable. However, ensure that `isLoading` provides the same functionality as `loading`.



<!-- This is an auto-generated comment by OSS CodeRabbit -->